client/lib/gsuite/gsuite-product-type.js

```diff
@@ -26,7 +9,7 @@ export function getProductSlug( productType ) {
  * @see emailManagementAddGSuiteUsers() in client/my-sites/email/paths.js
  */
 export function getProductType( productSlug ) {
```

Comment:
At some point we should probably name this function according to the file name, for consistency across the project.